docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1246,34 +1252,88 @@ function handleEffectsButton(btn, type, effect) {
   btn.innerText = title;
 }
 
-async function stopVideo() {
-  console.log('MeetingControls#stopVideo()');
-  try {
-    if (localMedia.cameraStream) {
-      localMedia.cameraStream.stop();
-    }
+async function stopStartVideo() {
+  if(stopVideoButton.innerText === 'Stop Video') {
+    console.log('MeetingControls#stopVideo()');
+    try {
+      if (localMedia.cameraStream) {
+        localMedia.cameraStream.stop();
+      }
 
-    console.log('MeetingControls#stopVideo() :: Successfully stopped video!');
+      console.log('MeetingControls#stopVideo() :: Successfully stopped video!');
```

Comment:
Shouldn't we set the innerText to "Start Video" here? Same comment for the stopStartAudio function

Reply:
The innerText is changed upon the 'stream-ended' event completion. The change set reflects the same. Please check. 

I should may be change the method name in the log from `stopVideo` to `startStopVideo` and the same for `audio` as well. Will update that.